bundles/debian/Containerfile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM docker.io/library/debian:11-slim as debbase
+
+ARG EXPECTED_VERSION
+ARG VERSION_SUFFIX
+
+WORKDIR /work
+
+COPY ./build.sh /work/build.sh
+
+RUN /work/build.sh $EXPECTED_VERSION $VERSION_SUFFIX /work/bundle.json
+
+FROM docker.io/library/golang:1.19 as gobuild
+
+WORKDIR /work
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+COPY main.go main.go
+
+RUN CGO_ENABLED=0 go build -o copypause main.go
```

Comment:
nit: `copypause` was a bit confusing to me on first pass. Can we call it `copyandpause` instead?